zebra-network/src/constants.rs

```diff
@@ -95,6 +95,14 @@ pub const EWMA_DEFAULT_RTT: Duration = Duration::from_secs(20 + 1);
 /// better peers when we restart the sync.
 pub const EWMA_DECAY_TIME: Duration = Duration::from_secs(200);
 
+/// Unix portion of OS error when the port attempting to be opened is already in use.
+#[cfg(unix)]
+pub const PORT_IN_USE_ERROR: &str = "already in use";
+
+/// Windows portion of OS error when the port attempting to be opened is already in use.
+#[cfg(not(unix))]
+pub const PORT_IN_USE_ERROR: &str = "one usage";
```

Comment:
@yaahc do you know how rustdoc and conditional compilation interact?

Does this work?
```suggestion
/// OS-specific error when the port attempting to be opened is already in use.
#[cfg(unix)]
pub const PORT_IN_USE_ERROR: &str = "already in use";
#[cfg(not(unix))]
pub const PORT_IN_USE_ERROR: &str = "one usage";
```

Reply:
That doesn't seem to work, but I was able to get something similar working:

```rust
/// OS-specific error when the port attempting to be opened is already in use.
#[cfg(unix)]
pub const PORT_IN_USE_ERROR: &str = if cfg!(unix) { "already in use" } else { "one usage" };
```

By default with constants the actual literal value is included in the rustdoc output, but it seems when we do something like this it only shows the type and the name, but not the value, which makes sense imo.